OSDOCS-20473 added new module (#114491)
openshift/openshift-docs: modules/zero-trust-manager-manually-delete-scc.adoc

// Module included in the following assemblies:
//
// * security/zero_trust_workload_identity_manager/zero-trust-manager-configuration.adoc

:_mod-docs-content-type: PROCEDURE
[id="zero-trust-manager-manually-delete-scc_{context}"]
= Manually delete the custom security context constraints

[role="_abstract"]

> Verify that the SPIFFE CSI driver is ready and its pods use the `privileged` security context constraint (SCC). Then delete the `spire-spiffe-csi-driver` custom SCC.

Reviewer: I wonder if this would be stronger if we start with the deletion, which is the job that needs doing. I am assuming this step is optional.

```suggestion
You can delete the `spire-spiffe-csi-driver` custom SCC when the SPIFFE CSI driver is ready and its pods are using the `privileged` security context constraint (SCC).
```

Author: Done

.Prerequisites

* You have upgraded {zero-trust-full} to 1.1.0 from the **OperatorHub** catalog.

.Procedure

. Confirm the SPIFFE CSI driver is ready by running the following command:
+
[source,terminal]
----
$ oc get spiffecsidriver cluster -o jsonpath='{range .status.conditions[*]}{.type}={.status}{"\n"}{end}'
----
+
The expected output includes `DaemonSetAvailable=True` and `Ready=True`.

. Confirm that the CSI `DaemonSet` is available by running the following command:
+
[source,terminal]
----
$ oc get ds spire-spiffe-csi-driver -n zero-trust-workload-identity-manager
----

> . Confirm CSI pods are running and using the `privileged` SCC by running the following commands:

Reviewer: ```suggestion
. Confirm the CSI pods are running and are using the `privileged` SCC by running the following commands:
```

Author: Done
